tests/py/dynamo/conversion/test_where_aten.py

```diff
@@ -76,6 +77,86 @@ def forward(self, condition):
             (condition,),
         )
 
+    # shape, min shape range, opt shape range, max shape range
+    @parameterized.expand(
+        [
+            (
+                "3d_condition_3d_xshape_3d_yshape",
+                (-1, -1, -1),
+                (1, 1, 1),
+                (1, 2, 3),
+                (3, 3, 3),
+                (-1, -1, -1),
+                (1, 1, 1),
+                (1, 2, 3),
+                (3, 3, 3),
+                (-1, -1, -1),
+                (1, 1, 1),
+                (1, 2, 3),
+                (3, 3, 3),
+            ),
+            (
+                "1d_condition_3d_xshape_2d_yshape",
+                (-1),
+                (1,),
+                (2,),
+                (4,),
+                (-1, -1, -1),
+                (1, 1, 1),
+                (3, 2, 2),
+                (3, 2, 4),
+                (
+                    -1,
+                    -1,
+                ),
+                (1, 1),
+                (2, 2),
+                (2, 4),
+            ),
+            (
+                "2d_condition_3d_xshape_2d_yshape",
```

Comment:
It would be great if you could also add a test case to verify broadcasting.

```
            (
                "name",
                (-1, -1),
                (4, 1),
                (4, 2),
                (5, 4),
                (-1, -1, -1),
                (1, 1, 1),
                (3, 1, 2),
                (3, 1, 4),
                (
                    -1,
                    -1,
                ),
                (1, 1),
                (1, 2),
                (1, 4),
            ),
```

Reply:
Thanks for suggestion. I replaced 2d_condition_3d_xshape_2d_yshape test with it